templates/dhcpd.conf-header.erb

```diff
@@ -15,6 +15,9 @@ log-facility <%= @logfacility %>;
 # ----------
 option domain-name "<%= @dnsdomain_real.first %>";
 option domain-name-servers <%= @nameservers.join(', ') %>;
+<% if @dnssearchdomains && @dnssearchdomains != [] -%>
```

Comment:
`if @dnssearchdomains && !@dnssearchdomains.empty?` is possibly better from a style perspective?

Also matches other templates.  See https://github.com/voxpupuli/puppet-dhcp/blob/master/templates/dhcpd.conf.ntp.erb#L2